docs/general/ad-filtering/create-own-filters.md

```diff
@@ -233,9 +233,7 @@ This modifier lets you narrow the rule coverage down to a specific application (
 
 > For Mac, you can find out the bundle ID or the process name of the app by viewing the respective request details in the Filtering log.
 
-> **Compatibility with different versions of AdGuard**
->
-> Starting with CoreLibs v1.12, the `$app` modifier for Windows is case-insensitive.
+> For Windows, starting with CoreLibs v1.12, the `$app` modifier for Windows is case-insensitive.
```

Comment:
Again, this is a part of the modifier definition, why a separate blockquote?

```
* Windows — use the process name, e.g. `chrome.exe`. Note, that the process name is case-insensitive starting with CoreLibs v1.12.
```

Reply:
Just for attention, similar to the Mac note. OK, I'll redo it.